package.json

```diff
@@ -25,11 +25,12 @@
   },
   "homepage": "https://github.com/cronofy/cronofy-node#readme",
   "dependencies": {
-    "lodash": "^3.10.1",
-    "rest": "^1.3.1"
+    "lodash": "^4.17.4",
```

Comment:
That's the upside, I appreciate the downside is it can be a bit trickier to maintain for all possible dependency combinations.